dist/profile/templates/buildmaster/casc/global-libraries.yaml.erb

```diff
@@ -0,0 +1,19 @@
+unclassified:
+  globalLibraries:
+    libraries:
+      - defaultVersion: "master"
+        implicit: true
+        includeInChangesets: false
+        name: "pipeline-library"
```

Comment:
you're missing the caching config from the other file?
```suggestion
        name: "pipeline-library"
        cachingConfiguration:
          refreshTimeMinutes: 180
```

Reply:
> [...] I've omitted the cache setting for now.

:)

Reply:
I'm resolving conversation to allow merging

Reply:
https://github.com/jenkins-infra/helpdesk/issues/2751 for tracking cache